web_src/js/features/aria.js

```diff
@@ -77,10 +77,11 @@ function attachOneDropdownAria($dropdown) {
     'aria-expanded': 'false',
   });
 
+  const isMenuVisible = () => $menu.hasClass('visible') || $menu.is('.animating.in');
```

Comment:
looks like the `aria-expand` stays true again. 
<img width="430" alt="截屏2023-03-13 13 43 37" src="https://user-images.githubusercontent.com/17645053/224617565-5dd70a2c-130d-4d11-9fa6-23081a4ed343.png">

This might be `isMenuVisible ` is not working properly, because it returns `true` even the when leaving the menu (By press 'tab' button). And I tried to replace this function with the old  `isMenuVisible` logic , which is

```js
const isMenuVisible = () => !$menu.is('.hidden') && !$menu.is('.animating.out');
```

And it works again, not very sure why though.

Reply:
Thank you, I made a mistake when processing "visible". I think it could be fixed by https://github.com/go-gitea/gitea/pull/23346/commits/c2086696c110e7b1f31e18cb33eccdbdafd9796e.

And I found some more problems in "dropdown init" code (eg: the inconsistent issue reaction menu & context menu init), I also fixed that problem and added more comments for "$.dropdown()" code. I think it's clear to developers now about how the "dropdown init" works.